TELCODOCS-368: Deploying an OCP cluster manually without Assisted Installer (regardless of use case) (#38068)
openshift/openshift-docs: modules/install-sno-monitoring-the-installation-manually.adoc

// This is included in the following assemblies:
//
// installing_sno/install-sno-installing-sno.adoc

[id="monitoring-the-installation-manually_{context}"]
= Monitoring the installation manually

If you created the ISO manually, use this procedure to monitor the installation.

.Procedure

. Monitor the installation:
+
[source,terminal]
----
$ openshift-install --dir=ocp wait-for install-complete
----
+

> The server will reboot several times, deploying the control plane.

Reviewer: ```suggestion
The server will reboot several times while it deploys the control plane.
```

> The server will reboot several times, deploying a control plane followed by a worker.

Reviewer 2: ```suggestion
The server will reboot several times, deploying the control plane.
```

> . After installation completes, check the environment (optional):

Reviewer: ```suggestion
. Optional: After installation completes, check the environment:
```

Author: Does the style guide call for this?

Reviewer 3: @johnwilkins Yes. It is in the IBM Style Guide.
